### ISSUE TITLE
S Pen Capabilities?

### ISSUE THREAD

**Issue by OGPandaGod** [Aliucord/plugin-requests#214]
### Discord Account

Panda God#0666

### Plugin Description

I think it would be cool if when you hovered an s pen over emojis and stuff if it would tell you what the emoji is without tapping

### Go into more detail...

It would be pretty cool if s pens or any other integrated styluses could have more capabilities to them

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by zt64** [comment]
I don't believe this is possible from just a plugin, as the S-Pen SDK requires adding two jars to the applications libraries

**Comment by RhythmLunatic** [comment]
It's impossible without recompiling the app because you have to declare S-Pen capabilities in the manifest. At least for Air Actions.
